tasks/build.py

```diff
@@ -269,8 +269,9 @@ def submit_job(job, submitted_jobs, build_env_cfg, ym, pr_id):
     cpu_target = '/'.join(job.arch_target.split('/')[1:])
 
     # Add a default time limit of 24h to the command if nothing else is specified by the user
-    all_opts = " ".join([build_env_cfg[SLURM_PARAMS], job.slurm_opts, '--export=ALL,CPU_TARGET=%s' % cpu_target])
-    if ("--time" in all_opts) or ("-t" in all_opts):
+    all_opts_str = " ".join([build_env_cfg[SLURM_PARAMS], job.slurm_opts, '--export=ALL,CPU_TARGET=%s' % cpu_target])
```

Comment:
I would remove the third argument `, '--export=ALL,CPU_TARGET=%' % cpu_target]`. This is not a configuration parameter that we expect to include a time limit. In a future version of the bot this code is probably going away anyway.